api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -424,116 +424,117 @@ uiConfiguration: >-
     },
     "criteriaConfigs": [{
       "type":"concept",
+      "id": "tanagra-conditions",
       "title":"Conditions",
       "defaultName":"Contains Conditions Codes",
-      "plugin": {
-        "columns": [
-          {"key":"concept_name","width":"100%","title":"Concept Name"},
-          {"key":"concept_id","width":120,"title":"Concept ID"},
-          {"key":"standard_concept","width":180,"title":"SourceStandard"},
-          {"key":"vocabulary_id","width":120,"title":"Vocab"},
-          {"key":"concept_code","width":120,"title":"Code"},
-          {"key":"person_count","width":120,"title":"Roll-up Count"}
-        ],
-        "hierarchyColumns": [
-          {"key": "concept_name", "width": "100%", "title": "Condition"},
-          {"key": "concept_id", "width": 120, "title": "Concept ID"},
-          {"key":"person_count","width":120,"title":"Roll-up Count"}
-        ],
-        "occurrence": "condition_occurrence",
-        "classification": "condition"
-      }
+      "columns": [
```

Comment:
Not for this PR, but should we be generating this from the list of entity attributes in the underlay? That might require additional properties of each attribute (e.g. display name, display width), but then the info would be inline instead of in a separate block.

Reply:
I'm not sure about this. One example where that doesn't work would be if there was more than one way to view an entity. Perhaps some elements can be inlines and some would remain separate?

Reply:
OK, let's revisit post-underlay overhaul.